packages/ses/src/compartment-shim.js

```diff
@@ -155,7 +289,7 @@ export const makeCompartmentConstructor = (
   nativeBrander,
 ) => {
   /** @type {CompartmentConstructor} */
-  function Compartment(endowments = {}, _moduleMap = {}, options = {}) {
+  function Compartment(endowments = {}, moduleMap = {}, options = {}) {
     if (new.target === undefined) {
       throw new TypeError(
         "Class constructor Compartment cannot be invoked without 'new'",
```

Comment:
I'm happy with this error behavior. But that makes me even more curious why you didn't to that for StaticModuleRecord above?

Reply:
Eventual consistency. I’m used to the pre-class prevailing precedent and only recently learned of `new.target`. I’ll keep this one. The one in `StaticModuleRecord` is getting deleted in an upcoming change.